app/svelte/src/server/options.ts

```diff
@@ -1,4 +1,4 @@
-import packageJson from '../../package.json';
+const packageJson = require('../../package.json');
```

Comment:
why?? @gaetanmaisse

Reply:
Just saw your comment. Still this seems weird to me. Any references I can look at?

Reply:
By setting `"rootDir": "./src"` we are telling TS compiler that all sources will be in `src` folder.  As `package.json` is not in `src` it can not be imported directly (i.e. at compile time) without throwing: `File '/Path/to/package.json' is not under 'rootDir' '/Path/To/App/src/'. 'rootDir' is expected to contain all source files.`

`require` will import the file at runtime so TS compiler will not try to load it and finally everyone is happy.